_posts/0000-01-02-outlawtorn49.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"Love and a cough 
cannot be concealed. 
Even a small cough. 
Even a small love."